tests/unit/waiter-manager-noop-test.ts

```diff
@@ -1,7 +1,8 @@
+import { _reset, buildWaiter, getWaiters, register } from 'ember-test-waiters';
 import { module, test } from 'qunit';
+
 import { DEBUG } from '@glimmer/env';
-import { _reset, buildWaiter, register, getWaiters } from 'ember-test-waiters';
-import NoopTestWaiter from 'ember-test-waiters/noop-test-waiter';
+import NoopTestWaiter from 'ember-test-waiters/build-waiter';
```

Comment:
This seems wrong? I don't think it is the default export of that file right? Maybe it isn't used at all?

Reply:
No it's not. Not sure how this isn't erroring out. Fixed now.